selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/constant/Infrastructure.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.selenium.core.constant;
+
+import com.google.inject.Singleton;
+
+@Singleton
+public enum Infrastructure {
+  docker,
```

Comment:
upper case please

Reply:
You are right that it is the constant and have to be in upper case.
But we are using `Infrastructure` enum to simplify injection from environment variable `CHE_INFRASTRUCTURE` like the follow:
```
@Inject
@Named("che.infrastructure")
private Infrastructure infrastructure;
```
Such injection is case sensitive, and `CHE_INFRASTRUCTURE` uses lower-case values like *docker*, *openshift* etc.

Reply:
I've asked about values, of enum not about the name of enum itself.

Reply:
We can transform the value of `CHE_INFRASTRUCTURE` variable to upper case in time of configuring CheSeleniumSuiteModule to make it possible to inject upper case values: Infrastructure.DOCKER, Infrastructure.OPENSHIFT etc.

Reply:
Not sure what do you mean. But lowercase enum values looks weird.